app/src/main/java/com/kylecombes/micrepeater/Setting.java

```diff
@@ -0,0 +1,44 @@
+package com.kylecombes.micrepeater;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.DisplayMetrics;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CompoundButton;
+import android.widget.ImageButton;
+import android.widget.Switch;
+import android.widget.TextView;
+
+public class Setting extends MainActivity {

```

Comment:
Should name the class `SettingActivity` and only extend `Activity` (not `MainActivity`). We don't want to extend `MainActivity` here because they are not the same.